plenum/common/ledger_manager.py

```diff
@@ -112,24 +112,35 @@ def request_CPs_if_needed(self, ledgerId):
         ledgerInfo.consistencyProofsTimer = None
         ledgerInfo.recvdCatchupRepliesFrm = {}
 
+    @staticmethod
+    def _missing_txns(ledger_info) -> Tuple[bool, int]:
+        ledger = ledger_info.ledger
+        if ledger_info.catchupReplyTimer is None:
+            return False, 0
+
+        end = getattr(ledger_info.catchUpTill, f.SEQ_NO_END.nm)
+
+        catchUpReplies = ledger_info.receivedCatchUpReplies
+        total_missing = (end - ledger.size) - len(catchUpReplies)
```

Comment:
What if end=30, ledger.size=20, and we have 10 receivedCatchUpReplies, but with seqNo from 1 to 10 (already processed). The method will return `total_missing == 0`, while it should be 10.